File: src/main/java/com/qa/persistence/domain/Character.java
package com.qa.persistence.domain;

import javax.persistence.Entity;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import javax.persistence.Table;

@Entity
@Table(name = "CHARACTER")
public class Character {
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Id
	private Long id;
	private String name;
	private String race;
	private String characterClass;
	private String level;
	private String strength;
	private String dexterity;
	private String constitution;
	private String intelligence;
	private String wisdom;
	private String charisma;
	
	


	

	public Character(String name, String race, String characterClass, String background, String level,
			String subClass, String spells, String strength, String dexterity, String constitution, String intelligence, String wisdom,
			String charisma) {
		super();
		this.name = name;
		this.race = race;
		this.characterClass = characterClass;
		this.level = level;
		this.strength = strength;
		this.dexterity = dexterity;
		this.constitution = constitution;
		this.intelligence = intelligence;
		this.wisdom = wisdom;
		this.charisma = charisma;

	}
	public Character() {

	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getRace() {
		return race;
	}

	public void setRace(String race) {
		this.race = race;
	}

	public String getCharacterClass() {
		return characterClass;
	}

	public void setCharacterClass(String characterClass) {
		this.characterClass = characterClass;
	}



	public String getLevel() {
		return level;
	}

	public void setLevel(String level) {
		this.level = level;
	}

	public String getStrength() {
		return strength;
	}

	public void setStrength(String strength) {
		this.strength = strength;
	}

	public String getDexterity() {
		return dexterity;
	}

	public void setDexterity(String dexterity) {
		this.dexterity = dexterity;
	}

	public String getConstitution() {
		return constitution;
	}

	public void setConstitution(String constitution) {
		this.constitution = constitution;
	}

	public String getIntelligence() {
		return intelligence;
	}

	public void setIntelligence(String intelligence) {
		this.intelligence = intelligence;
	}

	public String getWisdom() {
		return wisdom;
	}

	public void setWisdom(String wisdom) {
		this.wisdom = wisdom;
	}

	public String getCharisma() {
		return charisma;
	}

	public void setCharisma(String charisma) {
		this.charisma = charisma;
	}


	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}

}

File: src/Character.js
import React, { Component } from 'react';

class Character extends Component {
    render() {
        return (
            <div>
                 <div>
                    {this.props.data.name}
                </div>
                <div>
                    {this.props.data.race}
                </div>
                <div>
                    {this.props.data.characterClass}
                </div>
                 <div>
                    {this.props.data.level}
                </div>
                <div>
                    {this.props.data.strength}
                </div>
                <div>
                   {this.props.data.dexterity} 
                </div>
                  <div>
                    {this.props.data.constitution}
                </div>
                <div>
                    {this.props.data.intelligence}
                </div>
                <div>
                    {this.props.data.wisdom} 
                </div>
                 <div>
                    {this.props.data.charisma}
                </div>
                <br />
            </div>
        );
    }
}

export default Character;
